common/filter-utilities.js

```diff
@@ -0,0 +1,86 @@
+import {
+  isImageType,
+  isVideoType,
+  isAudioType,
+  isDocument,
+  isTwitterLink,
+  isYoutubeLink,
+  isTwitchLink,
+  isGithubLink,
+  isInstagramLink,
```

Comment:
Just a heads up: in the future instead of presets, what we will do is take a user's top saved link domains, and list them. For now we can leave it like this as a preview to see if we want to keep this.